### ISSUE TITLE
Replace usage of twisted.web.html.PRE in twistsed code

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7942]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#7942|
|Type|enhancement|
|Created|2015-06-17 06:15:53Z|
|Branch|https://github.com/twisted/twisted/tree/t.web.html-to-template-7942|

As mentioned in [#4948](https://github.com/twisted/twisted/issues/4948) twisted.web.html will die and we need to migrate code to something else... maybe twisted.web.templates

there are a few places using html.PRE and they can be refactored.

<details><summary>Searchable metadata</summary>

```
trac-id__7942 7942
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_t_web_html_to_template_7942 branches/t.web.html-to-template-7942
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1434521753244595 1434521753244595
changetime__1435587562118279 1435587562118279
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

(In [45083]) Branching to t.web.html-to-template-7942.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban removed owner|
|-|-|

I first tried to port current html.PRE usage to twisted.web.template, but I have no idea how to do it without a big refactoring.

This is why I moved it into utils._PRE and wrote missing test for the code using the html.PRE

I think that the new code is better than current trunk.

t.web.html is not tested, not maintained and with few features.

I think that users should use external libraries if they want to generate HTML tags.
 
We ask users to move away from t.web.html and use t.web.templates but as I found out, migration might involve a lot of changes.

Please check latest changes.. and let me know how do plan to process with t.web.html deprecation.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="sunu's avatar" src="https://avatars.githubusercontent.com/u/1142203?s=50" width="50" height="50">](https://github.com/sunu)<a name="note_3"></a>|@sunu commented|
|-|-|

+1. The changes look good to me.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set owner to @adiroiban|
|-|-|

LGTM. Buildbots look good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [45133]) Merge t.web.html-to-template-7942: Replace usage of twisted.web.html.PRE in twistsed code.

Author: adiroiban
Reviewers: _sunu_, hawkowl
Fixes: [#7942](https://github.com/twisted/twisted/issues/7942)
